docs/docs/overview/performance.md

```diff
@@ -6,103 +6,178 @@ This section analyzes the performance of Constellation.
+The [DC4as machine type](https://learn.microsoft.com/en-us/azure/virtual-machines/dasv5-dadsv5-series#dasv5-series) with four cores provides the following maximum performance:
+- 6400 (20000 burst) IOPS
+- 144 MB/s (600 MB/s burst) throughput
+However, the performance is bound by the capabilities of the 512 GiB Standard SSD size:
+- 500 (600 burst) IOPS
+- 60 MB/s (150 MB/s burst) throughput
+
+For Constellation on GCP and GKE, we ran the benchmark with Compute Engine Persistent Disk Storage [pd-standard](https://cloud.google.com/compute/docs/disks) of 400 GB size.
+Our N2D machine type with four cores and pd-standard provides the following [maximum performance](https://cloud.google.com/compute/docs/disks/performance#n2d_vms):
+- 15,000 write IOPS
+- 3,000 read IOPS
+- 240 MB/s write throughput
+- 240 MB/s read throughput
+
+The [`fio`](https://fio.readthedocs.io/en/latest/fio_doc.html) benchmark consists of several tests.
+We utilized [`Kubestr`](https://github.com/kastenhq/kubestr) to run `fio` in Kubernetes.
+The default test performs randomized access patterns that accurately depict worst-case I/O scenarios for most applications.
+
+The following `fio` settings were used:
+
+- No Cloud caching
+- No OS caching
+- Single CPU
+- 60 seconds runtime
+- 10 seconds ramp-up time
+- 10 GB file
+- IOPS: 4 KB blocks and 128 iodepth
+- Bandwidth: 1024 KB blocks and 128 iodepth
+
+You can find the `fio` test configuration [here](../../../.github/actions/e2e_benchmark/fio.ini).
+
+
+The results for "Rand Read" IOPS are as follows:
+
+![I/O read IOPS benchmark graph](../_media/benchmark_fio_read_iops.png)
+
+The results for "Rand Write" IOPS are as follows:
+
+![I/O write IOPS benchmark graph](../_media/benchmark_fio_write_iops.png)
 
-The K-Bench [`fio`](https://fio.readthedocs.io/en/latest/fio_doc.html) benchmark consists of several tests.
-We selected four different tests that perform asynchronous access patterns because we believe they most accurately depict real-world I/O access for most applications.
+The results for "Rand Read" bandwidth are as follows:
 
-The following graph shows I/O throughput in MiB/s (higher is better).
+![I/O read bandwidth benchmark graph](../_media/benchmark_fio_read_bw.png)
 
-![I/O benchmark graph](../_media/benchmark_io.png)
+The results for "Rand Write" bandwidth are as follows:
 
-Comparing Constellation on GCP with GKE, you see that Constellation offers similar read/write speeds in all scenarios.
+![I/O write bandwidth benchmark graph](../_media/benchmark_fio_write_bw.png)
 
-Constellation on Azure and AKS, however, partially differ. In read-write mixes, Constellation on Azure outperforms AKS in terms of I/O. On full-write access, Constellation and AKS have the same speed.
+Comparing Constellation on GCP with GKE, you see that Constellation has a similar read but 75% less write bandwidth.
+Constellation on Azure has 15% less read and 54% less write bandwidth compared to AKS.
```

Comment:
I can't reproduce these numbers from the graphs.

Reply:
Better now?